### PR TITLE
[CS0618] `ActorRefSinkSpec` `Sink.ActorRef<TIn>(IActorRef, object)` is obsolete

### PR DESCRIPTION
**Pull request by eaba**
## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).

### CODE REVIEW

File: src/core/Akka.Streams.Tests/Dsl/ActorRefSinkSpec.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ public ActorRefSinkSpec(ITestOutputHelper output) : base(output, StreamTestDefau
         [Fact]
         public void ActorRefSink_should_send_elements_to_the_ActorRef()
         {
-            Source.From(new[] { 1, 2, 3 }).RunWith(Sink.ActorRef<int>(TestActor, onCompleteMessage: "done"), Materializer);
+            Source.From(new[] { 1, 2, 3 }).RunWith(Sink.ActorRef<int>(TestActor, onCompleteMessage: "done", _ => ""), Materializer);
```
Comment (eaba) [inline src/core/Akka.Streams.Tests/Dsl/ActorRefSinkSpec.cs:39]: Changed into `Sink<TIn, NotUsed> ActorRef<TIn>(IActorRef actorRef, object onCompleteMessage, Func<Exception, object> onFailureMessage)`

File: src/core/Akka.Streams.Tests/Dsl/ActorRefSinkSpec.cs [hunk 2]
```diff
@@ -48,7 +48,7 @@ public void ActorRefSink_should_send_elements_to_the_ActorRef()
         public void ActorRefSink_should_cancel_a_stream_when_actor_terminates()
         {
             var fw = Sys.ActorOf(Props.Create(() => new Fw(TestActor)).WithDispatcher("akka.test.stream-dispatcher"));
-            var publisher = this.SourceProbe<int>().To(Sink.ActorRef<int>(fw, onCompleteMessage: "done"))
+            var publisher = this.SourceProbe<int>().To(Sink.ActorRef<int>(fw, onCompleteMessage: "done", _ => ""))
```
Comment (eaba) [inline src/core/Akka.Streams.Tests/Dsl/ActorRefSinkSpec.cs:51]: Changed into `Sink<TIn, NotUsed> ActorRef<TIn>(IActorRef actorRef, object onCompleteMessage, Func<Exception, object> onFailureMessage)`

**Review by eaba — COMMENTED** [review]
Changes

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**